### PR TITLE
Implemented watchlist. Adding to watchlist, seeing movies in watchlis…

### PR DESCRIPTION
**Pull request by Kendra1**
…t, removing movies from watchlist and marking movies as watched. Made sure user sees a message if watchlist is empty and that he cannot click two times on buttons for adding movies to watchlist and marking that the movies is watched. Made sure that list is updated after removing movie from watchlist

### CODE REVIEW

File: src/component/Movie.js [hunk 1]
```diff
@@ -89,6 +90,17 @@ class Movie extends Component {
                 >
                   Dislike!
                 </button>
+
+                <button
+                  // className="button1"
```
Comment (AlexSensei) [inline src/component/Movie.js]: brisi ako je visak :D

Comment (Kendra1) [reply src/component/Movie.js]: jesam, nego zaboravila da komitujem -.-

File: src/store/reducers/WatchlistReducer.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        ...state,
+        watchlist: state.watchlist.map(item =>
+          item.id === payload[0].id ? payload[0] : item
+        )
+      };
+    }
+    case UPDATE_WATCHLIST:
+      return {
+        ...state,
+        watchlist: state.watchlist.filter(item => item.id !== action.payload)
```
Comment (AlexSensei) [inline src/store/reducers/WatchlistReducer.js]: filter(({id})=> id !==action.payload)

File: src/store/sagas/WatchlistSagas.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+export function* deleteItem({ payload }) {
+  try {
+    const { data } = yield call(watchlistService.deleteItem, payload);
+
+    yield put(updateWatchlist(data));
+  } catch (error) {
+    console.log({ error });
+  }
+}
```
Comment (AlexSensei) [inline src/store/sagas/WatchlistSagas.js]: nidje lodera

**Pull request merged**